integrations/nginx/install.go

```diff
@@ -0,0 +1,349 @@
+			assetOpt = optionals.Some(a)
+		}
+	}
+	asset, present := assetOpt.Get()
+	if !present {
+		return unsupportedError
+	}
+
+	printer.Infof("Selected prebuilt module %s\n", asset.Name)
+
+	// Find the Nginx directory
+	destDir := args.DestDir
+	if destDir.IsNone() {
+		destDir = FindNginxModuleDir()
+	}
+	if destDir.IsNone() {
+		printer.Warningf("Can't identify directory for module to be installed.\n")
+	} else {
+		printer.Infof("Module will be installed in %v\n", destDir.GetOrDefault(""))
+	}
+
+	// Bail out if the user didn't actually ask for the download
+	if args.DryRun {
+		printer.Infof("Ready to download; skipping due to --dry-run flag.\n")
+		return nil
+	}
+
+	// Create a temporary directory for download and a shorter filename inside it.
+	tmpDir, err := os.MkdirTemp("", "akita-nginx-download")
+	if err != nil {
+		printer.Errorf("Can't create temporary directory for download: %v\n", err)
+		return downloadError
+	}
+	shortName := fmt.Sprintf("ngx_http_akita_module_%s.so", version)
+	downloadFile := filepath.Join(tmpDir, shortName)
+
+	// Delete the temporary directory on function exit, unless we explicitly decide to save it.
+	saveDir := false
+	defer func() {
+		if !saveDir {
+			os.RemoveAll(tmpDir)
+		}
+	}()
+
+	err = DownloadReleaseAsset(asset.ID, downloadFile)
+	if err != nil {
+		telemetry.Error("NGINX download asset", err)
+		return downloadError
+	}
+
+	// Move the file to its final location, if any.
+	if dir, ok := destDir.Get(); ok {
+		destPath := filepath.Join(dir, shortName)
+		err = os.Rename(downloadFile, destPath)
+		if err != nil {
+			printer.Errorf("Error moving module to the NGINX module directory: %v\n", err)
+			telemetry.Error("NGINX install module", err)
+			saveDir = true
+			return NewCopyError(downloadFile)
+		}
+
+		// Delete any existing symlink; expect failure if it doesn't exist.
+		err = os.Remove(filepath.Join(dir, "ngx_http_akita_module.so"))
+		if err != nil && !errors.Is(err, os.ErrNotExist) {
+			// Probably a permissions problem? Log but keep going.
+			printer.Infof("Can't remove old symbolic link: %v\n", err)
+			telemetry.Error("NGINX install module", err)
+		}
+
+		// Create a symlink with no version number, as recommended
+		err = os.Symlink(shortName,
+			filepath.Join(dir, "ngx_http_akita_module.so"))
+		if err != nil {
+			printer.Debugf("Error creating symlink: %v\n", err)
+			telemetry.Error("NGINX install module", err)
+			return symlinkError
+		}
+	} else {
+		saveDir = true
+		return NewCopyError(downloadFile)
+	}
+
+	printer.Infof("Module ngx_http_akita_module.so successfully installed!\n")
+	printer.Infof("To start using the NGINX module,\n" +
+		" 1. Add 'load_module ngx_http_akita_module.so' to the top of your configuration file.\n" +
+		" 2. Add 'akita_enable on;` to the locations handling the HTTP traffic you want to monitor.\n" +
+		" 3. Run `akita nginx capture --project <project name>` with the project name you have created in the Akita App.\n" +
```

Comment:
Does NGINX need to be reloaded or restarted to pick up its configuration change?

Reply:
Yes, added a step in  https://github.com/akitasoftware/akita-cli/pull/198/commits/134d8a7f7cc5e16e380e5d0684feee305e86df2b